.babelrc

```diff
@@ -1,3 +1,21 @@
 {
-  "presets": ["es2015", "stage-2", "react"]
+  "presets": ["react"],
+  "env": {
+    "commonjs": {
+      "plugins": [
```

Comment:
I think we can use the old `.babelrc` to fix `v2.0.0`. I think we need to use `latest` instead of `stage-2`.